inputs/text.vue

```diff
@@ -64,11 +68,8 @@
     },
     computed: {
       type() {
-        if (this.args.type === 'multi-line' || !this.args.type) {
-          return 'text';
-        } else {
-          return this.args.type;
-        }
+        return  this.args.type === 'multi-line' || !this.args.type ?
```

Comment:
what is this line even

Reply:
ah, I'm just turning this if into a ternary. eslint is stripping the `()` which I'd ordinarily keep in so it doesn't look confusing.